plugins/catalog/src/components/AboutCard/IconLinkVertical/IconLinkVertical.tsx

```diff
@@ -41,9 +46,24 @@ const useIconStyles = makeStyles({
 export function IconLinkVertical({
   icon = <LinkIcon />,
   href = '#',
+  disabled = false,
   ...props
 }: IconLinkVerticalProps) {
   const classes = useIconStyles();
+
+  if (disabled) {
+    return (
+      <Link
+        className={classnames(classes.link, classes.disabled)}
+        underline="none"
+        {...props}
+      >
+        {icon}
+        <span className={classes.label}>{props.label}</span>
+      </Link>
+    );
+  }
+
```

Comment:
Oh I meant to use one single return, where everything is the same except the `disabled` class is added if it's disabled. But, now I realise - you actually do NOT want to have the `<Link>` here at all if it's disabled, right? Isn't the point that it should not be clickable? :)

Reply:
yeah thats why it doesn't have a href, but now when you say it I can actually just make it another element and not a Link

Reply:
what would you prefer. a disabled link as it is now or change it to be a Typography with the link class? (since I want the same styles as the Link when its not disabled)

Reply:
Aha! Ok yeah well then it works anyway to keep using Link even though it's a bit unexpected perhaps.